lib/compress/zstd_compress.c

```diff
@@ -1531,15 +1531,6 @@ static void ZSTD_reduceIndex (ZSTD_CCtx* zc, const U32 reducerValue)
 
 /* See doc/zstd_compression_format.md for detailed format description */
 
```

Comment:
I'm surprised, I would expect `ZSTD_noCompressBlock()` to be useful ?
Is it not used for non-compressible blocks ?

Reply:
Not currently, it is inlined.

The two differences are we need `lastBlock` support, and it uses `MEM_writeLE32()` since it knows the block isn't empty.

We could extract it out into this function.

Reply:
Oh wow, this function is incorrect for the current format, it puts `bt_raw` in the lowest bit.

Reply:
So it's a (very old) pre-v1.0 leftover. 
Good thing this patch can detect that one !